rpm/sailfish-browser.spec

```diff
@@ -31,6 +31,7 @@ Requires: sailfish-browser-settings = %{version}
 Requires: qt5-plugin-imageformat-ico
 Requires: qt5-plugin-imageformat-gif
 Requires: qt5-plugin-position-geoclue
+Requires: mapplauncherd >= 4.1.17
```

Comment:
I think it's okay, mapplauncherd 4.1.17 was released in January according to https://github.com/nemomobile/mapplauncherd/releases.